src/Microsoft.Health.Fhir.Api/Configs/FhirServerConfiguration.cs

```diff
@@ -29,5 +29,7 @@ public class FhirServerConfiguration : IApiConfiguration
         public BundleConfiguration Bundle { get; } = new BundleConfiguration();
 
         public ThrottlingConfiguration Throttling { get; } = new ThrottlingConfiguration();
+
+        public ArtifactStoreConfiguration ArtifactStore { get; } = new ArtifactStoreConfiguration();
```

Comment:
>ArtifactStore [](http://example.com/codeflow?start=42&length=13)

If it's sole purpose to serve Convert operation, shouldn't it be part of Convert configuration?

Reply:
The previous config inside ConvertData is to serve convert operation only , but this service-level artifact registration will be used by convert data and anonymized export ([#AB81761(https://microsofthealth.visualstudio.com/Health/_backlogs/backlog/Compass/Stories/?workitem=81761)], and more scenarios requires external resources in the future